corehq/apps/builds/templates/builds/edit_menu.html

```diff
@@ -161,6 +179,9 @@ <h3 class="panel-title">Menu Options</h3>
                             <td>
                                 <input type="checkbox" value="superuser-only" data-bind="checked: superuser_only" />
                             </td>
+                            <td class='j2me_enabled'>
+                                <input type="checkbox" value="j2me_enabled" data-bind="checked: j2me_enabled" />
```

Comment:
This is slightly off from the spec, but in an important way.

> **Include a J2ME Enabled column** in [the build menu UI](https://www.commcarehq.org/builds/edit_menu/); a simple column that contains a checkmark icon if the build is j2me-enabled and is blank otherwise. This is for debugging purposes and to potentially reduce support burden of any confusion that arises from the split.

So you want it to look something like

<img width="340" alt="screen shot 2016-07-14 at 7 58 13 pm" src="https://cloud.githubusercontent.com/assets/137212/16859505/608a6d8c-49fd-11e6-8bc0-bb418a29c14d.png">

with a checkmark icon that's not editable. (Since it's tied to the Version you pick and not independent, it'd be bad for the user to be able to edit it directly.) You can do that with something like

```html
                            <td class='j2me_enabled'>
                                <!--ko if: j2me_enabled-->
                                <i class="fa fa-check" data-bind="visible: j2me_enabled"></i>
                                <!--/ko-->
                            </td>
```

Looks like you already took care of setting to the correct value when you change the value of Version dropdown (where I commented above), so this might be the only change you have to make.